data/ecosystems/s/scroll.toml

```diff
@@ -484,21 +838,42 @@ url = "https://github.com/crackeryo/ethonline2023"
 [[repo]]
 url = "https://github.com/crisgarner/zkret-attestations"
 
+[[repo]]
+url = "https://github.com/cristianchaparroa/pocket"
+
+[[repo]]
+url = "https://github.com/crossScale/xScale"
+
+[[repo]]
+url = "https://github.com/cruzocards/cruzo-contracts"
+
 [[repo]]
 url = "https://github.com/crypto-hackers/ethglobal-tokyo"
 
 [[repo]]
 url = "https://github.com/cryptofin-ethdenver/diing-contract"
 
+[[repo]]
+url = "https://github.com/CryptoSnipper/scroll-dev-ai-bot"
+
+[[repo]]
+url = "https://github.com/cryptotwilight/3BD"
+
 [[repo]]
 url = "https://github.com/cryptotwilight/kuala-bond"
 
+[[repo]]
+url = "https://github.com/crystal-lang-tools/scry"
```

Comment:
This repo doesn't look applicable to scroll.

Reply:
Removed.